pkg/operator/ceph/cluster/nodedaemon/exporter.go

```diff
@@ -308,3 +308,11 @@ func generateExporterEnvVar() corev1.EnvVar {
 
 	return env
 }
+
+func ExporterIsHost(cephCluster cephv1.CephCluster) bool {
```

Comment:
Since the method is only called in the same package, it should be lower case
```suggestion
func exporterIsHost(cephCluster cephv1.CephCluster) bool {
```

Reply:
done